README.md

```diff
@@ -191,6 +191,7 @@ USAGE: consul-templaterb [[options]]
     -M, --debug-memory-usage         Display messages when RAM grows
     -T, --trim-mode=trim_mode        ERB Trim mode to use (- by default)
     -R, --sig-reload=reload_signal   Signal to send to next --exec command on reload (NONE supported), default=HUP
+    -s, --restart                    restart the --exec command on each configuration change(if your process do not handle hot-reload)    
```

Comment:
already use
'-r', '--retry-delay=<min_duration>'

Reply:
In that case, I propose to remove the short option
See https://ruby-doc.org/stdlib-2.7.1/libdoc/optparse/rdoc/OptionParser.html - it doesn't need to be specified.
Better not put any to avoid bringing confusion